web-services-app/src/test/java/edu/unc/lib/boxc/web/services/rest/DownloadImageControllerIT.java

```diff
@@ -14,7 +14,6 @@
 import edu.unc.lib.boxc.search.solr.models.ContentObjectSolrRecord;
 import edu.unc.lib.boxc.web.common.services.SolrQueryLayerService;
 import edu.unc.lib.boxc.web.services.processing.DownloadImageService;
-import edu.unc.lib.boxc.web.services.processing.ImageServerProxyService;
```

Comment:
was this just an unused import?

Reply:
Yeah, it got cleaned up by intellij when I was moving things around